velox/exec/HashAggregation.cpp

```diff
@@ -233,7 +237,7 @@ RowVectorPtr HashAggregation::getOutput() {
     // allow for memory reuse.
     input_ = nullptr;
 
-    flushPartialOutputIfNeed();
+    resetPartialOutputIfNeed();
```

Comment:
would calling this lead to incorrect result if the intermediate state is flushed?
cc @mbasmanova

Reply:
@shiyu-bytedance I'm not sure I understand the questions. Would you provide some context? I'm seeing random questions of diffs from you and wondering what you are trying to achieve and how we can help.